src/core_atmosphere/Registry.xml

```diff
@@ -397,6 +397,7 @@
         <packages>
                 <package name="mp_kessler_in" description="parameterization of kessler microphysics."/>
                 <package name="mp_thompson_in" description="parameterization of Thompson cloud microphysics."/>
+                <package name="mp_thompson_aers_in" description="parameterization of scale-aware Thompson cloud microphysics."/>
```

Comment:
Should this be called "parameterization of **aerosol**-aware Thompson cloud microphysics"?